src/Shared.elm

```diff
@@ -50,6 +50,16 @@ type alias Model =
     , connection : Connection
     , rightSidebarExtended : Bool
     , controlMenu : Bool
+    , musicHover : Bool
```

Comment:
While this works, I would recommend to have a custom `type` that represents what is hovered. As there is only ever one thing that can be hovered in this case. 
This has a lot more possible states, that might happen and are unwanted. Basically all states that are have more then one `true`.

It would be nicer with `type LeftSidebarMenuHover = None | Music | Movies | ...`

Reply:
I have checked the model and application, at a time only one icon can be hovered and no other states are possible. Do I still need to make any changes to the model?

Reply:
I would recommend it, as it vastly improves the design of your store and thus the handling. Also, while you think it's not possible, eliminating it at the source will make it impossible to happen.

Reply:
Yeah I think you are right. I will start working on it straight away.